dispatch/static/manager/src/js/pages/EventAudit/index.js

```diff
@@ -0,0 +1,181 @@
+import React from 'react'
+import R from 'ramda'
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group'
+import { connect } from 'react-redux'
+import DocumentTitle from 'react-document-title'
+import { replace } from 'react-router-redux'
+
+import eventsActions from '../../actions/EventsActions'
+
+import EventCard from './EventCard'
+import EventAuditPagination from './EventAuditPagination'
+
+require('../../../styles/components/event_audit.scss')
+
+const PER_PAGE = 15
+
+const PENDING_QUERY = {
+  pending: 1,
+  limit: PER_PAGE
+}
+
+class EventAuditPage extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      count: 0
```

Comment:
How hard would it be to control this in the store? I know it works this way, but it breaks out of the redux architecture and seems kinda hard to follow. I might be missing something though

Ideally the events list that we keep in the store would stay up to date, and any time an event goes from `is_submission=True` to `is_submission=False`, that change would be reflected. Then you can simply count up all of the events with `is_submission=True` and use that as your count